demisto_sdk/commands/common/tools.py

```diff
@@ -1554,20 +1554,24 @@ def arg_to_list(arg: Union[str, List[str]], separator: str = ",") -> List[str]:
     return [arg]
 
 
-def is_v2_file(current_file, check_in_display=False):
```

Comment:
please make sure this method is not used anywhere else.

Reply:
Made sure